subprojects/packagefiles/jbig2dec/src/jbig2dec.def

```diff
@@ -0,0 +1,13 @@
+EXPORTS
+; Public API.
+jbig2_complete_page
+jbig2_ctx_free
+jbig2_ctx_new_imp
+jbig2_data_in
+jbig2_global_ctx_free
+jbig2_make_global_ctx
+jbig2_page_out
+jbig2_release_page
+; Private, but used by the jbig2dec executable.
```

Comment:
that's somewhat interesting. I'm guessing this doesn't use visibility attributes either.

Reply:
Nope, the shared library exports everything…